src/main/java/core/basesyntax/strategy/impl/PostingStrategy.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitMovement;
+import core.basesyntax.strategy.PostingGoods;
+import core.basesyntax.strategy.Strategy;
+
+public class PostingStrategy implements Strategy {
+    public PostingGoods getPostingAccordingMovement(FruitMovement movement) {
+        switch (movement.getType()) {
+            case BALANCE:
+                return new PostingBalanceOfFruits();
+            case SUPPLY:
+                return new PostingSupplyOfFruit();
+            case PURCHASE:
+                return new PostingPurchaseOfFruit();
+            case RETURN:
+            default:
+                return new PostingReturnOfFruit();
+        }
+    }
```

Comment:
світч-кейси погано масштабуються, краще використовувати мапу
її можна створювати на початку, і потім передавати в конструктор стратегії і робити наступне:
```suggestion
    public PostingGoods getPostingAccordingMovement(MovementType type) {
        return handlers.get(type);  
    }
```